src/consts.ts

```diff
@@ -91,6 +91,7 @@ function hasKey<O>(obj: O, key: keyof any): key is keyof O {
 
 const LIGHT_UP_CHINATOWN_TIER_2_MIN = 45;
 const LIGHT_UP_CHINATOWN_TIER_3_MIN = 150;
+const MEGA_GAM = 'mega_gam';
```

Comment:
What is the purpose of creating a constant? Could we use the enum instead?

Reply:
ah that's right. will change